src/game/etj_progression_tracker.h

```diff
@@ -43,19 +43,29 @@ class ProgressionTrackers {
     char *setIf;
     char *increment;
     char *incrementIf;
+    char *bitIsSet;
+    char *bitNotSet;
+    char *bitSet;
+    char *bitReset;
   };
 
   static const int MaxProgressionTrackers = 50;
+  static ETJump::ProgressionTrackers::ProgressionTrackerKeys ParseTrackerKeys();
+
   struct ProgressionTracker {
     ProgressionTracker() {
-      for (auto &v : equal) v = -1;
-      for (auto &v : notEqual) v = -1;
-      for (auto &v : greaterThan) v = -1;
-      for (auto &v : lessThan) v = -1;
-      for (auto &v : set) v = -1;
-      for (auto &v : setIf) v = -1;
-      for (auto &v : increment) v = 0;
-      for (auto &v : incrementIf) v = 0;
+      std::fill_n(equal, MaxProgressionTrackers, -1);
+      std::fill_n(notEqual, MaxProgressionTrackers, -1);
+      std::fill_n(greaterThan, MaxProgressionTrackers, -1);
+      std::fill_n(lessThan, MaxProgressionTrackers, -1);
+      std::fill_n(set, MaxProgressionTrackers, -1);
+      std::fill_n(setIf, MaxProgressionTrackers, -1);
+      std::fill_n(increment, MaxProgressionTrackers, 0);
+      std::fill_n(incrementIf, MaxProgressionTrackers, 0);
+      std::fill_n(bitIsSet, MaxProgressionTrackers, -1);
+      std::fill_n(bitNotSet, MaxProgressionTrackers, -1);
+      std::fill_n(bitSet, MaxProgressionTrackers, -1);
+      std::fill_n(bitReset, MaxProgressionTrackers, -1);
```

Comment:
Could rather use the `ProgressionTrackerValueNotSet` constant here as opposed to `-1` where applicable.